core/src/main/java/greencity/controller/UserController.java

```diff
@@ -388,15 +392,20 @@ public ResponseEntity<Boolean> checkIfTheUserIsOnline(
      */
     @Operation(summary = "Get user profile statistics by id")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
-        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
+            @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "403", description = HttpStatuses.FORBIDDEN)
     })
     @GetMapping("/{userId}/profileStatistics/")
     public ResponseEntity<UserProfileStatisticsDto> getUserProfileStatistics(
-        @Parameter(description = "Id of current user. Cannot be empty.") @PathVariable @CurrentUserId Long userId) {
+            @PathVariable Long userId
+            , @Parameter(description = "Id of current user. Cannot be empty.") @CurrentUserId Long currentUserId) {
```

Comment:
I'm not sure, but maybe it would be better instead to add Long currentUserId better add @CurrentUser UserVO currentUser. And then do something like if (!currentUser.getId().equals(userId))? User has already logged in, so we can get information about him but if use currentUserId as a parameter I can pass any ID here, even not the current user's.
Think about it.